src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/model/TickClock.java

```diff
@@ -17,6 +17,7 @@
 
 import lombok.Getter;
 
+// TODO - del
```

Comment:
Why not to delete this class right now?

Reply:
It is not related to the current change. This is my own reminder, it should not appear in final commit.